src/persist/src/indexed/cache.rs

```diff
@@ -81,19 +83,8 @@ impl<B: Blob> BlobCache<B> {
         self.blob.lock()?.close()
     }
 
-    /// Returns the batch for the given key, blocking to fetch if it's not
-    /// already in the cache.
-    pub fn get_unsealed_batch(&self, key: &str) -> Result<Arc<BlobUnsealedBatch>, Error> {
-        {
-            // New scope to ensure the cache lock is dropped during the
-            // (expensive) get.
-            if let Some(entry) = self.unsealed.lock()?.get(key) {
-                self.metrics.blob_read_cache_hit_count.inc();
-                return Ok(entry.clone());
-            }
-            self.metrics.blob_read_cache_miss_count.inc();
-        }
-
+    /// Synchronously fetches the batch for the given key.
```

Comment:
Should this method (and the one for traces) be public still? The behaviour changed (it doesn't try and go to the cache anymore) and it's only used as an internal helper of the async version right now.

Reply:
Good point, done